README.rst

```diff
@@ -0,0 +1,114 @@
+django-component-views
+=============================
+
+.. image:: https://img.shields.io/pypi/v/edx-proctoring.svg
+    :target: https://pypi.python.org/pypi/edx-proctoring/
+    :alt: PyPI
+
+.. image:: https://travis-ci.org/edx/edx-proctoring.svg?branch=master
+    :target: https://travis-ci.org/edx/edx-proctoring
+    :alt: Travis
+
+.. image:: http://codecov.io/github/edx/edx-proctoring/coverage.svg?branch=master
+    :target: http://codecov.io/github/edx/edx-proctoring?branch=master
+    :alt: Codecov
+
+.. image:: https://img.shields.io/pypi/pyversions/edx-proctoring.svg
+    :target: https://pypi.python.org/pypi/edx-proctoring/
+    :alt: Supported Python versions
+
+.. image:: https://img.shields.io/github/license/edx/django-component-views.svg
+    :target: https://github.com/edx/edx-proctoring/blob/master/LICENSE.txt
+    :alt: License
+
+This is the exam proctoring subsystem for the Open edX platform.
+
+Overview
+--------
+
+Proctored exams are exams with time limits that learners complete while online
+proctoring software monitors their computers and behavior for activity that
+might be evidence of cheating. This Python library provides the proctoring
+implementation used by Open edX.
+
+Documentation
```

Comment:
I've added documentation for the basic make commands. I decided against documenting all of the Transifex commands beyond the help command.